webpack/assets/javascripts/react_app/redux/reducers/notifications/index.js

```diff
@@ -0,0 +1,34 @@
+import {
+  GET_NOTIFICATIONS,
+  SET_NOTIFICATIONS_DRAWER_OPENED,
+  SET_EXPANDED_GROUP,
```

Comment:
can you name this with a notification_prefix? its hard to filter the notifications actions with a page with many other events?